config/config.sample.json

```diff
@@ -1,12 +1,24 @@
 {
-  "default_server_config": {
-    "m.homeserver": {
-      "base_url": "http://synapse.localhost:8008",
-      "server_name": "synapse.localhost"
+  "default_server_config":{
+    "m.homeserver":{
+      "base_url":"https://mydomain.com",
+      "server_name":"mydomain.com"
     }
   },
-  "features": {
-    "feature_use_device_session_member_events": true
+  "livekit": {
+    "livekit_service_url":"https://livekit-jwt.mydomain.com"
   },
-  "eula": "https://static.element.io/legal/online-EULA.pdf"
```

Comment:
I *think* the EULA field is currently a required field so the example should match. However, it will be made optional soon as it is not used when Element Call is running as an embedded widget.